website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+To see why this is useful, imagine trying to build a responsive layout with variably sized cards, 10px apart from each other, hugging the edges of the parent container. Trying to accomplish this layout with child margins can be tricky.
+
+The following shows a layout where we start by giving each child `margin: 10`:
+
+![Two diagrams. On the left it shows a skeleton of an app with three boxes that have margin around them cause the boxes to have margin around all sides. On the right, the same diagram is shown highlighted to demonstrate the margin on all sides.](/blog/assets/FlexboxGapBefore.png)
+
+Margins are applied uniformly to the edges of all children and don’t collapse under flexbox, giving us spacing at the exterior of the cards, and double the space on the interior compared to what we wanted. We can get around this by applying non-uniform margins, using negative margins on the parent, halving our intended spacing, etc, but it can be made much easier.
+
+With flex gap, this layout can be achieved by setting `gap: 10` for a 10px gap between the interior of the cards:
+
+![[Two diagrams. On the left it shows a skeleton of an app with three boxes that have margin only on the inner sides and not the outer sides of the boxes due to the Flexbox gap property. On the right, the same diagram is shown highlighted to demonstrate the margin only on the inner sides.](/blog/assets/FlexboxGapAfter.png)
+
+For more information on Flexbox gaps, see [_https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps_](https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps).
+
+## Web-inspired props for accessibility, styles, and events
+
+:::note
+
+For more background on aligning React Native to web standards, check out this [_proposal_](https://github.com/necolas/discussions-and-proposals/blob/reduce-fragmentation/proposals/0000-reduce-fragmentation.md) and [_related discussion_](https://github.com/react-native-community/discussions-and-proposals/pull/496).
+:::
+
+This release includes a number of new props inspired by web standards to align React Native’s APIs across many platforms. These new props are purely additive so there are no expected migrations or change of behavior for equivalent accessibility, behavior, or style props.
+
+For any new prop alias introduced, if there is an existing prop with a different name and both are specified, the new alias prop value will take precedence. For example, this release adds a `src` prop alias for `source` on the Image component to align with the `src` prop on web. If both `src` and `source` are provided, the new `src` prop will be used.
+
+#### Accessibility
+
+We introduced ARIA props as alias to existing React Native accessibility props.
+
+These props now exist on all core components of React Native:
+
+- `aria-label`
```

Comment:
Did these land with dashes in the name? For some reason I thought they landed with camel case like `ariaLabel`

Reply:
Oh good call out, just checked and they were landed with dashes.

Reply:
Yeah, including the dash matches web https://reactjs.org/docs/accessibility.html#wai-aria